lib/rby/core/url_launcher.dart

```diff
@@ -14,12 +16,19 @@ final launcherProvider = Provider<UrlLauncher>(
         ref.read(mediaPreferencesProvider).openLinksExternally;
 
     try {
-      await launchUrl(
-        Uri.parse(url),
-        mode: shouldOpenExternally
-            ? LaunchMode.externalApplication
-            : LaunchMode.inAppWebView,
-      );
+      if (shouldOpenExternally) {
```

Comment:
@robertodoering: I'd like to move this file out of the `rby` package because it has dependencies to the `components` and now even the `core` library. What do you think?

Reply:
Yes!